Resources/Prototypes/Adventure/_CyberneticGenesis/LilaKebab/meal_recipes.yml

```
@@ -0,0 +1,17 @@
+#Pizzas TODO: contruction graph based pizza
+- type: microwaveMealRecipe
+  id: RecipeLilaKebabShaurma
+  name: шаурма ли ла кебаб
+  result: FoodMealLilaKebab
+  time: 30
+  group: Savory
+  reagents:
+    BbqSauce: 5
+    Ketchup: 5
+  solids:
+    FoodDoughFlat: 3

```

```suggestion
    FoodDoughFlat: 3
```
3 лепёшки как-будто слишком много, хватит и одной